app/assets/stylesheets/themes/cultural_repository.scss

```diff
@@ -1,53 +1,68 @@
-.cultural_repository  {
+.cultural__repository {
```

Comment:
I get that the linter told you to change this but you need to leave it as cultural_repository. This selector is generated by a controller.